Rename Datadog tracing library to Datadog SDK (#35358)
DataDog/documentation: content/en/tests/setup/java.md

Default configuration values work well in most cases.

> However, if there is a need to fine-tune the SDK's behavior, [Datadog Tracer configuration][3] options can be used.

Reviewer: Two issues here: "fine-tune" is on the word list (use "customize" instead), and the link text "Datadog Tracer configuration" wasn't updated.

```suggestion
However, to customize the SDK's behavior, [Datadog SDK configuration][3] options can be used.
```

> However, if there is a need to fine-tune the SDK's behavior, [Datadog Tracer configuration][3] options can be used.

Reviewer: Two issues here: "fine-tune" is on the word list (use "customize" instead), and the link text "Datadog Tracer configuration" wasn't updated.

```suggestion
However, to customize the SDK's behavior, [Datadog SDK configuration][3] options can be used.
```
